app/src/test/java/com/willowtree/vocable/FakeCategoriesUseCase.kt

```diff
@@ -72,7 +88,11 @@ class FakeCategoriesUseCase : ICategoriesUseCase {
         _categories.update { categories ->
             categories.map {
                 if (it.categoryId == categoryId) {
-                    it.copy(hidden = hidden)
+                    when(it) {
+                        is Category.StoredCategory -> it.copy(hidden = hidden)
+                        is Category.PresetCategory -> it.copy(hidden = hidden)
+                        is Category.Recents -> it.copy(hidden = hidden)
```

Comment:
ah I was thinking the few instances that looks like this were a little smelly but looks like this is not a problem in the real impl because of  `withHidden(...)` and friends. 

fine to keep this as-is imo since we're deprecating this bad boy

Reply:
Ah shoot, I forgot about the `withHidden()`, you just reminded me why I created that to begin with!

Agreed, let's keep this as-is since it's going to get deleted anyway, and this approach isn't really harming anyone. Agreed though, it is a little smelly.